js/foundation.abide.js

```diff
@@ -385,6 +427,11 @@ class Abide extends Plugin {
         validator = $el.attr('data-validator'),
         equalTo = true;
 
+    // skip validation if disabled
```

Comment:
Why do we need this here as well? Don't we skip validating inputs below in validateForm before we would ever get to this?

Reply:
both are public methods, meaning you may call `validateInput` without triggering `validateForm`

furthermore Abide itself calls only `validateInput` on change of an input
(`validateForm` gets only called on submit)

that's why I've to skip validation in both methods

Reply:
within `validateForm` I'm resetting this.formnovalidate which I use to recognize if the submit was triggered by a formnovalidate submit (otherwise it would affect future events)